python/plugins/grassprovider/tests/testdata/grass_algorithms_imagery_tests.yaml

```diff
@@ -328,7 +328,9 @@ tests:
         type: raster
     results:
       imaginary:
-        hash: 94249384dd8b6019f0024501bc9a093cba9dd025c183d3fb46d77027
+        hash:
```

Comment:
why changes here?

Reply:
Newer grass is giving slightly different results for the algorithms, so it's to get the tests passing on newer versions

Reply:
(the hash list is a list of acceptable values)

Reply:
Ok, thanks. That's what I thought, but I wanted confirmation.